atomic_reactor/plugins/post_group_manifests.py

```diff
@@ -0,0 +1,112 @@
+        self.group = group
+        self.goarch = goarch
+        self.registries = deepcopy(registries)
+
+    def get_worker_manifest(self, worker_data):
+        worker_digests = worker_data['digests']
+        worker_manifest = []
+
+        for registry, registry_conf in self.registries.items():
+            if not registry.startswith('http://') and not registry.startswith('https://'):
+                registry = 'https://' + registry
+
+            registry_noschema = urlparse(registry).netloc
+            for push_conf_registry in self.workflow.push_conf.docker_registries:
+                if push_conf_registry.uri == registry_noschema:
+                    break
+            else:
+                continue
+
+            insecure = push_conf_registry.insecure
+            auth = None
+            secret_path = registry_conf.get('secret')
+            if secret_path:
+                self.log.debug("registry %s secret %s", registry_noschema, secret_path)
+                dockercfg = Dockercfg(secret_path).get_credentials(registry_noschema)
+                try:
+                    username = dockercfg['username']
+                    password = dockercfg['password']
+                except KeyError:
+                    self.log.error("credentials for registry %s not found in %s",
+                                   registry_noschema, secret_path)
+                else:
+                    self.log.debug("found user %s for registry %s", username, registry_noschema)
+                    auth = requests.auth.HTTPBasicAuth(username, password)
+
+            digest = worker_digests[0]['digest']
+            tag = worker_digests[0]['tag']
+            repo = tag.split(':')[0]
+            for image in self.workflow.tag_conf.images:
+                image_tag = image.to_str(registry=False)
+
+                url = '{}/v2/{}/manifests/{}'.format(registry, repo, digest)
+
+                response = requests.head(url, verify=not insecure, auth=auth)
+                response = requests.get(url, verify=not insecure, auth=auth)
+
+                image_manifest = json.loads(response.text)
+                image_manifest['tag'] = image_tag
+
+                url = '{}/v2/{}/manifests/{}'.format(registry, repo, image_tag)
+                response = requests.put(url, data=json.dumps(image_manifest),
+                                        verify=not insecure, auth=auth)
+
+                worker_manifest.append(image_manifest)
+
+        return worker_manifest
+
+    def run(self):
+        if self.group:
+            raise NotImplementedError('group=True is not supported in group_manifest')
+        grouped_manifests = []
+
+        build_result = self.workflow.buildstep_result.get(OrchestrateBuildPlugin.key)
+        all_annotations = build_result.annotations
+        for platform in all_annotations['worker-builds']:
+            if self.goarch.get(platform, platform) == 'amd64':
```

Comment:
That's right. http://osbs.readthedocs.io/en/latest/multiarch.html#group-manifests